drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/builders/IndexedParametersValueBuilder.java

```diff
@@ -29,40 +29,40 @@
         implements
         ParameterizedValueBuilder {
 
-    private static final Pattern delimiter = Pattern.compile( "(.*?[^\\\\])(,|\\z)" );
+    private static final Pattern delimiter = Pattern.compile("(.*?[^\\\\])(,|\\z)");
 
     private final String template;
 
     private final List<String> parameters = new ArrayList<String>();
 
     private List<List<DTCellValue52>> values = new ArrayList<List<DTCellValue52>>();
 
-    public IndexedParametersValueBuilder( final String template,
-                                          final ParameterUtilities parameterUtilities,
-                                          final Part part ) {
-        this.template = parameterUtilities.convertIndexedParametersToTemplateKeys( template,
-                                                                                   part );
-        this.parameters.addAll( parameterUtilities.extractTemplateKeys( this.template ) );
+    public IndexedParametersValueBuilder(final String template,
+                                         final ParameterUtilities parameterUtilities,
+                                         final Part part) {
+        this.template = parameterUtilities.convertIndexedParametersToTemplateKeys(template,
+                                                                                  part);
+        this.parameters.addAll(parameterUtilities.extractTemplateKeys(this.template));
     }
 
-    public void addCellValue( final int row,
-                              final int column,
-                              final String value ) {
-        final List<String> cellVals = split( value );
+    public void addCellValue(final int row,
+                             final int column,
+                             final String value) {
+        final List<String> cellVals = split(value);
         final List<DTCellValue52> rowValues = new ArrayList<DTCellValue52>();
-        for ( int parameterIndex = 0; parameterIndex < getParameters().size(); parameterIndex++ ) {
-            final String cv = cellVals.size() > parameterIndex ? cellVals.get( parameterIndex ) : "";
-            rowValues.add( new DTCellValue52( cv ) );
+        for (int parameterIndex = 0; parameterIndex < getParameters().size(); parameterIndex++) {
+            final String cv = cellVals.size() > parameterIndex ? cellVals.get(parameterIndex).trim() : "";
```

Comment:
```trim()``` split values (e.g. "18, 24" becomes "18" and "24" both of which can be converted to numerical values).